packages/clarity-visualize/src/interaction.ts

```diff
@@ -99,8 +145,148 @@ export function pointer(event: Interaction.PointerEvent, iframe: HTMLIFrameEleme
         case Data.Event.TouchCancel:
             p.style.background = `url(${touchIcon}) no-repeat left center`;
             break;
+        case Data.Event.MouseMove:
+            p.style.background = `url(${pointerIcon}) no-repeat left center`;
+            points.push({time: event.time, x: data.x, y: data.y});
+            drawTrail(points[points.length-1].time, overlay(iframe));
+            break;
         default:
             p.style.background = `url(${pointerIcon}) no-repeat left center`;
             break;
     }
-}
\ No newline at end of file
+}
+
+function drawClick(doc: Document, x: number, y: number): void {
+    let click = doc.createElement("DIV");
```

Comment:
do we want to create a new element for every click ?

Reply:
Yes, so we can visualize those individually.